scripts/1

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Need to rename the file with a proper name, which is a basic requirement for every file to be checked in.

Reply:
Is this file duplicate of scripts/cvpc.json?

Reply:
fixed in 3bfa97f